.github/workflows/check_signed_new.yaml

```diff
@@ -0,0 +1,31 @@
+# Workflow to check if a user has correctly signed the CLA
+
+name: Check Signed
+
+# on:
+#   issue_comment:
+
+jobs:
+  check-signed:
+    name: Check CLA signed
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          repository: dfinity/public-workflows
+      - name: Install Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - name: Install Dependencies
+        run: pip install -q -r requirements.txt
+        shell: bash
+      - name: Check CLA issue
+        run: |
+          export PYTHONPATH="$PWD/reusable_workflows/"
+          python reusable_workflows/check_cla/check_cla_pr.py
```

Comment:
The path seems to be wrong.

Reply:
It is correct on this branch https://github.com/dfinity/public-workflows/tree/remove-action-files/reusable_workflows/check_cla